src/workerd/jsg/jsg.h

```diff
@@ -1932,10 +1936,19 @@ class Lock {
   // eliminate direct use as much as possible.
 
   v8::Local<v8::Value> v8Undefined();
+  v8::Local<v8::Value> v8Null();
 
   v8::Local<v8::Value> v8Error(kj::StringPtr message);
   v8::Local<v8::Value> v8TypeError(kj::StringPtr message);
 
+  bool v8Set(v8::Local<v8::Object> obj, V8Ref<v8::String>& name, Value& value);
```

Comment:
Should `v8Set()` return `void` instead of `bool`? I notice none of the call sites pay attention to the result, but people adding new calls might be confused about whether they are supposed to do something with it. V8 returns a bool, but what does it mean? If it's always true (or always false), let's just assert it and return `void`.